pkg/fees/btsTxFees.ts

```diff
@@ -0,0 +1,12 @@
+ /**
+ * Provides Bitshares transaction fees.
+ *
+ * @returns - Fixed BTS transaction fee.
+ */
+export async function getFeesBTS(): Promise<number> {
```

Comment:
This shouldn't be async.

Also why do we have fees for bts? We don't need them. the BTS library / blockchain does it all automatically.

Reply:
Deleted BTS tx fees files